src/en/space-station-14/departments/science/proposals/scientific-research-proposal.md

```diff
@@ -0,0 +1,35 @@
+# Scientific research proposal
+
+| Designers     | Implemented | GitHub Links |
+|---------------|-------------|--------------|
+| chromiumboy   | :x: No      | TBD          |
+
+## Problem statement
+At present the science department faces several key gameplay issues when it comes to unlocking items through scientific research. These include (but are not limited to):
+-	A large number of research projects that can be considered ‘filler’. These projects unlock either very niche or very common place items on the station; a choice between unlocking magboots, hydroponic trays, or buildable research disk terminals is not compelling for the player. 
+-	Due to the high proportion of filler projects at lower research tiers, typically one project will be seen as the ‘optimal’ choice to research (generally either in the industrial or experiment disciplines), leading to many niche items being ignored in the early to mid stages of the shift when they would be most useful, as well as a sense of sameness in technology unlocking despite the research projects being presented at random.
```

Comment:
```suggestion
- High proportion of 'filler' tech at lower research tiers leads to considering such technologies as stepping stones towards some final project - some t3 tech usually. This means that picking tech does not feel really important to player, lowering involvement, giving dissatisfaction that work was done but result is not useful. This also means that techs that are low tier but belong to discipline with non desider t3 will most likely will not be researched during the early to mid game stages of the shift when they would be most useful and can be in high demand.
```

Reply:
The idea is, kinda, that people are not having fun themselves, by spending earned points on it AND they are not letting other people have fun, because what other department want 'it is not in discipline that was targeted'. which is double negative.